lib/screens/settings/about.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:expandable/expandable.dart';
+
+class AboutPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text("About")),
+      body: SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(2.0),
+          child: Column(
+            children: <Widget>[
+              Card(
+                semanticContainer: true,
+                clipBehavior: Clip.antiAliasWithSaveLayer,
+                child: Column(
+                  children: <Widget>[
+                    Image.asset("assets/images/mentorship_system_logo.png"),
+                    Padding(
+                      padding: const EdgeInsets.all(8.0),
+                      child: ExpandablePanel(
+                        header: Text(
+                          "Who are we?",
+                          style: TextStyle(
+                            fontWeight: FontWeight.bold,
+                            fontSize: 22,
+                          ),
+                        ),
+                        collapsed: Text("AnitaB.org is a global nonprofit organization"),
+                        expanded: Text(
+                            "AnitaB.org is a global nonprofit organization based in Belmont, California. Founded by computer scientists Anita Borg and Telle Whitney, the institute's primary aim is to recruit, retain, and advance women in technology."),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              Card(
+                child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: ExpandablePanel(
+                    header: Text(
+                      "What is the Mentorship System",
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        fontSize: 22,
+                      ),
+                    ),
+                    collapsed: Text("Mentorship System is a cross platform application"),
+                    expanded: Text(
+                      "Mentorship System is a cross platform application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period of time.",
+                    ),
+                  ),
+                ),
+              ),
```

Comment:
I'd extract code building these 3 cards into separate build functions (e.x `_buildGeneralInfoCard`, `_buildContributingCard` etc). That way, it'll be much clearer.